addons/account/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,195 @@
+account      / account.account.tag      / active (boolean)              : NEW 
+account      / account.account.template / group_id (many2one)           : NEW relation: account.group
+account      / account.bank.statement   / website_message_ids (one2many): NEW relation: mail.message
+# NOTHING TO DO
+
+account      / account.bank.statement.line / journal_entry_ids (one2many)  : relation is now 'account.move.line' ('account.move')
+# NOTHING TO DO, it is handled in "statement_line_id" updates below
+
+account      / account.cash.rounding    / account_id (many2one)         : NEW relation: account.account
+account      / account.cash.rounding    / name (char)                   : NEW required: required
+account      / account.cash.rounding    / rounding (float)              : NEW required: required
+account      / account.cash.rounding    / rounding_method (selection)   : NEW required: required, selection_keys: ['DOWN', 'HALF-UP', 'UP'], req_default: function
+account      / account.cash.rounding    / strategy (selection)          : NEW required: required, selection_keys: ['add_invoice_line', 'biggest_tax'], req_default: function
+account      / account.fiscal.position.template / auto_apply (boolean)          : NEW 
+account      / account.fiscal.position.template / country_group_id (many2one)   : NEW relation: res.country.group
+account      / account.fiscal.position.template / country_id (many2one)         : NEW relation: res.country
+account      / account.fiscal.position.template / sequence (integer)            : NEW 
+account      / account.fiscal.position.template / state_ids (many2many)         : NEW relation: res.country.state
+account      / account.fiscal.position.template / vat_required (boolean)        : NEW 
+account      / account.fiscal.position.template / zip_from (integer)            : NEW 
+account      / account.fiscal.position.template / zip_to (integer)              : NEW 
+account      / account.full.reconcile   / exchange_partial_rec_id (many2one): DEL relation: account.partial.reconcile
+account      / account.group            / code_prefix (char)            : NEW 
+account      / account.group            / name (char)                   : NEW required: required
+account      / account.group            / parent_id (many2one)          : NEW relation: account.group
+account      / account.group            / parent_left (integer)         : NEW 
+account      / account.group            / parent_right (integer)        : NEW 
+account      / account.invoice          / access_token (char)           : NEW 
+account      / account.invoice          / activity_ids (one2many)       : NEW relation: mail.activity
+account      / account.invoice          / cash_rounding_id (many2one)   : NEW relation: account.cash.rounding
+account      / account.invoice          / refund_invoice_ids (one2many) : NEW relation: account.invoice
+# NOTHING TO DO
```

Comment:
Can you note here that refund_invoice_ids is the One2many counterpart of an existing Many2one field?